ideascube/utils.py

```diff
@@ -9,6 +13,19 @@ def __get__(self, cls, owner):
         return self.fget.__get__(None, owner)()
 
 
+def get_server_name():
```

Comment:
We agreed to leave it for now, and work on caching if performance ever become a problem.